libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -799,13 +831,22 @@ void AC_Avoid::adjust_velocity_polygon(float kP, float accel_cmss, Vector2f &des
         }
     }
 
+    // Active avoidance
```

Comment:
maybe we can improve this comment?  "// safe location is between min and max limits".. again I'm not sure